lab-taylor/index.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const getFiles = require(__dirname + '/lib/getFiles');
+const files = ['one.txt', 'two.txt', 'three.txt'];
+const path = './data/';
+
+const fullPath = files.map((file)=>{
+  return `${path}${file}`;
+});
+
+getFiles.readFiles(fullPath, (err, data) => {
+  if (err) {console.log(err);}
```

Comment:
Just note, you don't need the { } when declaring a one line if statement.
if (err) console.log(err);
would function just the same :)